### ISSUE TITLE
monitor command overflow?

### ISSUE THREAD

**Issue by staze** [arachnidlabs/reload-pro#63]
Issuing a monitor command via terminal, with 10000, or 60000 (10 seconds, or 60 seconds) results in a monitor value of 1 (or so) being set (terminal quickly overflows). monitor 9999' does not cause this same overflow.

**Comment by uwehermann** [comment]
Yup, works fine for me as well (firmware 1.11). Tested 10000 and 60000.
